codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/RustWriter.kt

```diff
@@ -387,6 +387,7 @@ class RustWriter private constructor(
         fun factory(debugMode: Boolean): Factory<RustWriter> = Factory { fileName: String, namespace: String ->
             when {
                 fileName.endsWith(".toml") -> RustWriter(fileName, namespace, "#", debugMode = debugMode)
+                fileName.endsWith(".py") -> RustWriter(fileName, namespace, "#", debugMode = debugMode)
```

Comment:
Can we do this customization in Python namespace?

Reply:
I don't think so.. To me it is reasonable to have it here, same goes for `toml` for example, but I would like to hear the SDK team thoughts..